airbyte-integrations/bases/standard-test/sample_files/standard_test_config.yml

```diff
@@ -0,0 +1,6 @@
+connector_image: airbyte/source-hubspot:0.1.0
```

Comment:
How are we going to inject the version?

Reply:
to be honest I don't know, this way of configuring standard test suite was in original requirements, 
@sherifnada please give me a hand here :)
offtopic: it is relatively easy to override tag/version from custom command-option (something like `pytest --connector="airbyte/source-hubspot:0.1.0"`)